### DISCUSSION TITLE
No options for workflow notifications in Android app

### DISCUSSION THREAD

**Discussion by dmwyatt** [community/community#44792] [Mobile] [answered]
### Select Topic Area

Product Feedback

### Body

I saw [the blog post](https://github.blog/changelog/2023-01-17-push-notifications-for-actions-on-mobile/) about workflow notifications for the app, but...there doesn't seem to be any options for workflow notifications in the Android app.

I did make sure there were no updates available for the app.  Play Store says it was last updated on Jan 10th, 2023.

![Screenshot_20230119-095101](https://user-images.githubusercontent.com/163359/213489885-8410df48-d1e1-480e-b406-12b6322c770f.png)

![Screenshot_20230119-095120](https://user-images.githubusercontent.com/163359/213489904-bc622642-33d7-486b-a4bf-4d5b455fd2e7.png)

**Comment by stevepopovich** [comment, marked as answer]
Hi @dmwyatt. We are actually rolling this out this out in a phased manor. By EOD Friday, all users should receive the update.

We definitely should not have released those communications before all users have received the update. Our bad. Sorry for the confusion!
